code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -43,7 +43,8 @@
 	if(internal_type && true_spawn)
 		internal = new internal_type(src)
 	apply_status_effect(STATUS_EFFECT_CRUSHERDAMAGETRACKING)
-	add_trait(TRAIT_NO_TELEPORT, MEGAFAUNA_TRAIT)
+	add_trait(TRAIT_NO_TELEPORT)
```

Comment:
revert this

Reply:
Tested it locally. Did not work without this change for some unknown reason.